internal/application/application.go

```diff
@@ -38,6 +38,7 @@ func EnumerateApplication(ctx context.Context, oktaConfig *okta.Configuration) (
 			return &methodokta.ApplicationReport{}, err
 		}
 		allApps = append(allApps, apps...)
+		time.Sleep(1000 * time.Millisecond)
```

Comment:
@sean-hacker It doesn't send back a status code. Under the hood its a 429 but the SDK hides it. I can get 200s back from the resp variable but when it bugs out the resp is nil and just an err. I could definitely do a try statement. This is from their docs for contrext

> Default rate limits
> API endpoint throttling limits have been created to protect the Okta service (in Production and Preview tenants) from load spikes or service interruptions caused by submitted requests, whether unintentional or as a Denial of Service attack. This strategy has been adopted across the industry in many SaaS applications and platforms to maintain consistent service levels for customers. Requests that hit the rate limits return a 429 Too Many Requests HTTP status code.
> 
> Public applications are aggressively rate-limited to prevent abuse and require primary authentication to be successfully completed before releasing any metadata about a user.
> 
> End-user rate limit
> Okta limits the number of requests from the Okta user interface to 40 requests per user per 10 seconds per endpoint. This rate limit protects users from each other, and from other API requests in the system.

Reply:
Okay I think I put together a pretty good solution. This will prevent failures too when CLIs are chained together